src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1990,6 +1990,19 @@ smart_objects::SmartObjectSPtr MessageHelper::CreateNegativeResponse(
   return std::make_shared<smart_objects::SmartObject>(response_data);
 }
 
+smart_objects::SmartObjectSPtr
+MessageHelper::CreateOnServiceStatusUpdateNotification(
+    const hmi_apis::Common_ServiceType::eType service_type) {
+  auto on_status_update_notification = CreateHMINotification(
+      hmi_apis::FunctionID::BasicCommunication_OnServiceUpdate);
+
+  auto& msg_ref = (*on_status_update_notification);
+
+  msg_ref[strings::msg_params][hmi_notification::service_type] = service_type;
```

Comment:
@mked-luxoft Why do you use msg_ref, but do not use on_status_update_notification?